### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Danie-O**

### CODE REVIEW

File: modules/network/variables.tf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    description = "Default Route from and to the internet"
+    type        = string
+    default     = "0.0.0.0/0"
+}
+
+variable "tenancy" {
+    default = "default"
+}
+
+variable "true" {
```
Comment (Mel-Zedeks) [inline modules/network/variables.tf]: i dont think you need to have a variable for something that will not change. if this value will change, then give it a better name than "true"

File: terraform.tfstate.backup
Comment (Mel-Zedeks) [inline terraform.tfstate.backup:1]: add to ignore file

File: terraform.tfstate
Comment (Mel-Zedeks) [inline terraform.tfstate:1]: add to ignore file

**Review by Mel-Zedeks — CHANGES_REQUESTED** [review]
i think you need to explain the code to us if possible

File: terraform.tfstate
Comment (Danie-O) [reply terraform.tfstate:1]: Done

File: terraform.tfstate.backup
Comment (Danie-O) [reply terraform.tfstate.backup:1]: Done

File: modules/network/variables.tf (hunk 1 above)
Comment (Danie-O) [reply modules/network/variables.tf]: Since the true value can be specified inline and does not change, I took out the true variable.

**Review by Mel-Zedeks — APPROVED** [review]

**Pull request merged**